src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random randomNumber = new Random();
+
+    private final ColorSupplier randomColor = new ColorSupplier();
+
+    public Ball getrandomball() {
+        Ball randomball = new Ball(randomColor.getRandomColor(),randomNumber.nextInt(100));
+        return randomball;
```

Comment:
```suggestion
        return new Ball(randomColor.getRandomColor(),randomNumber.nextInt(100));
```
Also make 100 a constant. Read checklist more carefully 
<img width="1036" alt="Screen Shot 2022-11-03 at 7 55 50 PM" src="https://user-images.githubusercontent.com/90692308/199798768-b8c371d1-6b79-41bc-bc0e-a0eb6cc9108b.png">
<img width="1051" alt="Screen Shot 2022-11-03 at 7 57 35 PM" src="https://user-images.githubusercontent.com/90692308/199798865-f80f0453-62ad-4c33-bc7c-9648325f11ab.png">